yarn.lock

```diff
@@ -113,6 +113,28 @@
     lodash "^4.17.5"
     to-fast-properties "^2.0.0"
 
+"@shiftkey/prebuild-install@5.2.4":
```

Comment:
I should cut a release to get https://github.com/atom/node-keytar/pull/163 onto NPM because I don't need this forked package to be immortalized any more 🙈

Reply:
[`4.4.1`](https://github.com/atom/node-keytar/releases/tag/v4.4.1) is up on NPM - please upgrade to that as it doesn't need my fork of `prebuild-install`.